machines/vic/certificate.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
Why we are writing our own? We could easily use https://golang.org/src/crypto/tls/generate_cert.go? And if we keen to using this can we at least make sure that we are doing _everything_ that generate_cert.go does - with proper attributions?